jena-core/src/main/java/org/apache/jena/shared/impl/PrefixMappingImpl.java

```diff
@@ -49,8 +50,8 @@ public PrefixMappingImpl()
         { 
         // ConcurrentHashMaps protects against breaking each datastructure
         // but does not protect against inconsistency.
-        prefixToURI = new ConcurrentHashMap<>();
-        URItoPrefix = new ConcurrentHashMap<>(); 
+        prefixToURI = new ConcurrentSkipListMap<>();
+        URItoPrefix = new ConcurrentSkipListMap<>();
         }
```

Comment:
The order of entries in a PrefixMapping is not guaranteed. This part of the contract of for `PrefixMapping`.

There are other implementations of `PrefixMapping`.

Where is this a problem?

Reply:
I understand. Thanks for pointing it out.